configs/fsdp.py

```diff
@@ -13,7 +13,7 @@ class fsdp_config:
     sharding_strategy: ShardingStrategy = ShardingStrategy.FULL_SHARD
     checkpoint_type: StateDictType = StateDictType.SHARDED_STATE_DICT  # alternatively can use SHARDED_STATE_DICT save one file per rank, and can resize the world-size.
     fsdp_activation_checkpointing: bool=True
-    pure_bf16: bool = True
+    pure_bf16: bool = False
```

Comment:
What is the reason for changing the default to False?

Reply:
I realized we are adding --pure_bf16 in our command, if we leave this to True and then user set --use_fp16 this one would be in effect by default as well, so it might be better to default to false and enable it as needed.